src/data/faq.json

```diff
@@ -376,6 +379,217 @@
                             }
                         ]
                     },
+                    {
+                        "title": "Warning without TAN",
+                        "id": "warning_without_tan",
+                        "indexed": true,
+                        "accordion": [
+                            {
+                                "title": "Information about warning without TAN",

```

Comment:
"warn without TAN" is OK in its short form, however if you expand this with "about" then you need to write "a TAN" otherwise it is grammatically jarring

"Information about warning without a TAN" would be better.

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/3322/commits/f2b4444eace8ae8143537742d5f8e4f971a82b60, thanks!